### ISSUE TITLE
XHTML doctype in HTML reports, but not valid xml

### ISSUE THREAD

**Issue by twanvl** [ndmitchell/hlint#646]
The html reports have xhtml doctype and a xhtml+xml content type, but the file is not valid xml. The reason is the < operator in the javascript code.

I suggest changing to a HMTL 5 doctype tag.
```html
<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="utf-8">
 ...
```

Running the report through the w3c html validator (https://validator.w3.org/) also reports some other errors.

**Comment by ndmitchell** [comment]
Thanks for the report. I switched to the `HTML` doctype. However, getting rid of all the validation errors requires not using `width` for the div in the body, which unfortunately is not trivial (simply moving it to CSS has totally different semantics). I'd welcome a patch, or I'll get to it at some point in the future.
